r/R/arrow-info.R

```diff
@@ -139,7 +139,8 @@ arrow_with_json <- function() {
 some_features_are_off <- function(features) {
   # `features` is a named logical vector (as in arrow_info()$capabilities)
   # Let's exclude some less relevant ones
-  blocklist <- c("lzo", "bz2", "brotli", "substrait")
+  # jemalloc is only included because it is sometimes disabled in our build process
+  blocklist <- c("lzo", "bz2", "brotli", "substrait", "jemalloc")
```

Comment:
This might be a little paranoid / silly, but we disable jemalloc in our own builds, and honestly if only jemalloc is off, but nothing else is, that's generally ok, so I would rather be less noisy so long as folks can read parquet